docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-bootstrap-switches-ref.adoc

```diff
@@ -0,0 +1,246 @@
+[id='kie-server-bootstrap-switches-ref']
+= Bootstrap Switches
+
+The {KIE_SERVER} accepts a number of bootstrap switches (system properties) to configure the behavior of the server.
+
+.Bootstrap Switches for Disabling {KIE_SERVER} Extensions
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.drools.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the BRM support (for example rules support).
```

Comment:
BRM ? We probably should not use this abbreviation anymore

Reply:
changed to Decision Manager